.rubocop.yml

```diff
@@ -140,3 +140,12 @@ Style/NegatedIf:
 
 Style/SafeNavigation:
   Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: true
+
+Style/RedundantFreeze:
+  Enabled: true
+
+Style/MutableConstant:
+  Enabled: false
```

Comment:
Remove this - this should be enabled. The error this is generating is legit - see #17

Reply:
As I understand this rule is for the .freeze. But if you add a string like # frozen_string_literal: true .freeze becomes redundant.

Reply:
Again, read my comments on #17. frozen_string_literal only freezes strings. Not other objects, including objects that contain strings. This style (properly) captures all of those - we should care about more than just strings.

Reply:
Gotcha.

Reply:
The other changes look good, I think we just want to remove this now.